src/functions.js

```diff
@@ -324,6 +324,36 @@ var _ = Mavo.Functions = {
 
 	len: text => str(text).length,
 
+	/**
+     * Search if a group, collection, or primitive contains needle
+	 * @returns Boolean if a haystack of object or primitive is passed
+	 * @returns Array of booleans if a haystack of array is passed
+     */
+    contains: (haystack, needle) => {
+		var ret = false;
+		if ($.type(haystack) === "object") {
+			for (var property in haystack) {
+				ret = _.contains(haystack[property], needle);
+				if ($.type(ret) === "array") {
+					ret = Mavo.Functions.or(ret);
+				}
+				if (ret) { 
+					return true; 
```

Comment:
Why this? Won't the `return ret` in the end suffice?

Reply:
If ret is false it shouldn't be returned and terminate the function yet since there might be a match later in the object.